packages/icon/src/react/icon-transformer.js

```diff
@@ -50,6 +67,9 @@ module.exports = fileNames
       buildComponentFile(filePath, { componentName })
     )
 
-    acc[id] = outputFileName
+    acc[id] = require(`./icons/${outputFileName}`)
+
     return acc
   }, {})
+
+module.exports = { ...temp }
```

Comment:
what is the reasoning for exporting a shallow clone of temp vs just keeping what was on old line 37?

Reply:
left over debugging. will remove